deno-import-map.json

```diff
@@ -0,0 +1,6 @@
+{
+  "imports": {
+    "netlify:edge": "https://edge-bootstrap.netlify.app/v1/index.ts",
+    "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev/server-build"
```

Comment:
These should go into `.vscode/deno_resolve_npm_imports.json` I think?

Reply:
See my comment below https://github.com/remix-run/remix/pull/3104/files#r877403976

Reply:
This file is in the root of the **project**, so these imports should either be in the project's `.vscode/settings` _or_ you should move it to the correct place.